### ISSUE TITLE
Princess reset when I jumped over a fence.

### ISSUE THREAD

**Issue by Bumpa123** [the-hyjal-project/bugtracker#241]
**Describe Your Issue**

I went to kill princess in brackwell pumpkin Patch and I jumped over a fence which made her evade and then reset. 
**Steps To Reproduce**

1. Pull princess
2. Run to fence 
3. Jump over it

Item: https://classic.wowhead.com/item=
Quest: https://classic.wowhead.com/quest=88/princess-must-die
Spell: https://classic.wowhead.com/spell=

**Expected Behavior**

Either they should sprint towards me over the fence or follow me around, whatever behaviour works just not reset.

**Comment by GM-Dave-THP** [comment]
I am not able to replicate this, tested the fences in the area and the NPC runs around the fence. Was it a certain fence this happened with?

**Comment by Bumpa123** [comment]
It was the fence on the right side of the Little house there. Not sure how to explain it. The house where the low lvl warlock quest is. If you imagine the field being a Square it was at the upper right part. I attacked princess and she hit me pretty hard so I wanted to test if I could hop back and forth over it  and kill her with fireblast. First jump was fine, she ran around towards the forest. Then I did a second jump and that’s where she got stuck just staring at me. One of her helpers hit me from a long distance tho. I continued standing there and eventually she reset.

Not sure if this is helpfule at all 😊




Skickades från E-post<https://go.microsoft.com/fwlink/?LinkId=550986> för Windows 10

Från: Dave<mailto:notifications@github.com>
Skickat: den 17 januari 2021 23:00
Till: the-hyjal-project/bugtracker<mailto:bugtracker@noreply.github.com>
Kopia: Bumpa123<mailto:battseeyon@hotmail.com>; Author<mailto:author@noreply.github.com>
Ämne: Re: [the-hyjal-project/bugtracker] Princess reset when I jumped over a fence. (#241)


I am not able to replicate this, tested the fences in the area and the NPC runs around the fence. Was it a certain fence this happened with?
